metadata_manager.py

```diff
@@ -60,25 +60,25 @@ def get_all_server(self, local):
     def clear_metadata(self):
         self.cursor.execute('DELETE FROM KnownServer')
         self.cursor.execute('DELETE FROM FileMap')
-        self.cursor.execute('DELETE FROM Stats')
+        self.cursor.execute('DELETE FROM Connections')
         self.conn.commit()
 
     # Adds a concurrent request of a uuid
     def add_concurrent_request(self, uuid):
-        self.cursor.execute('SELECT connections FROM Stats WHERE uuid=?', (uuid,))
+        self.cursor.execute('SELECT connections FROM Connections WHERE uuid=?', (uuid,))
         result = self.cursor.fetchone()
         if result is not None:
             current_connection = int(result[0]) + 1
-            self.cursor.execute('UPDATE Stats SET connections=? WHERE uuid=?', (current_connection, uuid))
+            self.cursor.execute('UPDATE Connections SET connections=? WHERE uuid=?', (current_connection, uuid))
```

Comment:
Wait this is a little strange. The way it's set up, if you run `get_concurrent_request`, the response will always be 1